docs/enforce-https-rule-gh-pages.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path')
+const fs = require('fs');
+
+/**
+ * A recursive function to find *.html files and replace their content,
+ * starting from the provided targetPath argument.
+ * @param {string} targetPath
+ */
+function processHtmlFilesFromPath(targetPath) {
+  const htmlFilter = '.html';
+
+  if (!fs.existsSync(targetPath)) {
+    console.log('Directory does not exist: ', targetPath);
+    return;
+  }
+
+  const files = fs.readdirSync(targetPath);
+
+  for (let i = 0; i < files.length; i++) {
+    const filePath = path.join(targetPath, files[i]);
+    const stat = fs.lstatSync(filePath);
+
+    if (stat.isDirectory()) {

```

Comment:
Why do we need to change http to https? We don't do this now and the documentation is correctly served.
also isn't there a configuration that will enable HUGO to automatically convert the URLs?

Reply:
TL;DR
Fixed.

Informatively:
- Fixed by changing the `baseURL` url to be with the **https://** protocol. Now Hugo generates the links to the resources with https://.
- I couldn't find any option but I guess this was not the corrent thing to search for because it is simply this^.
- We have done the fix manually and it is currently served like that but haven't noticed it because we are testing locally with **localhost** as a baseURL and not directly seeing changes in the [public documentation](https://CoherentLabs.github.io/GameUIComponents) link.